inspire_records/api/literature.py

```diff
@@ -24,8 +24,16 @@
 
 from __future__ import absolute_import, division, print_function
 
+
 from .base import InspireRecord
+from ..pidstore.api import PidStoreLiterature
 
 
 class LiteratureRecord(InspireRecord):
```

Comment:
Where are methods for citations?

Reply:
not there yet, just pid

Reply:
we have to create some sort of query builder, just throwing the functions there doesn't look ideal